<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Tạo Câu Hỏi Trắc Nghiệm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                stop:0 rgba(135, 206, 250, 255), stop:1 rgba(255, 182, 193, 255));
}

QLabel {
    color: #2c3e50;
    font-weight: bold;
}

QTextEdit, QSpinBox, QLineEdit {
    background-color: white;
    border: 2px solid #3498db;
    border-radius: 8px;
    padding: 8px;
    font-size: 12px;
}

QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #21618c;
}

QGroupBox {
    font-weight: bold;
    font-size: 14px;
    color: #2c3e50;
    border: 2px solid #3498db;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px 0 5px;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">
font-size: 18px;
color: #2c3e50;
font-weight: bold;
text-align: center;
       </string>
      </property>
      <property name="text">
       <string>Nhập nội dung để tạo câu hỏi trắc nghiệm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="inputTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Nhập nội dung tại đây...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="numQuestionsLabel">
        <property name="styleSheet">
         <string notr="true">
font-size: 14px;
color: #2c3e50;
font-weight: bold;
         </string>
        </property>
        <property name="text">
         <string>Số câu hỏi:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numQuestionsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Tạo Câu Hỏi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="questionsGroupBox">
      <property name="title">
       <string>Câu Hỏi Trắc Nghiệm</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="questionsTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="answersGroupBox">
      <property name="title">
       <string>Nhập Đáp Án Của Bạn</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="answersHelpLabel">
         <property name="styleSheet">
          <string notr="true">
font-size: 12px;
color: #7f8c8d;
font-style: italic;
          </string>
         </property>
         <property name="text">
          <string>Nhập đáp án theo thứ tự câu hỏi, cách nhau bằng dấu phẩy (ví dụ: A, B, C, D, A)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="answersLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLineEdit" name="answersLineEdit">
           <property name="placeholderText">
            <string>Nhập đáp án của bạn (A, B, C, D, A)...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gradeButton">
           <property name="styleSheet">
            <string notr="true">
background-color: #27ae60;
           </string>
           </property>
           <property name="text">
            <string>Chấm Điểm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="resultsGroupBox">
      <property name="title">
       <string>Kết Quả Chấm Điểm</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="resultsTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="userLabel">
        <property name="styleSheet">
         <string notr="true">
font-size: 12px;
color: #2c3e50;
font-weight: bold;
         </string>
        </property>
        <property name="text">
         <string>Chưa đăng nhập</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="styleSheet">
         <string notr="true">
background-color: #e74c3c;
         </string>
        </property>
        <property name="text">
         <string>Đăng Xuất</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
